lib/modules/platform/types.ts

```diff
@@ -270,7 +270,7 @@ export interface Platform {
 
 export interface PlatformScm {
   isBranchBehindBase(branchName: string, baseBranch: string): Promise<boolean>;
-  isBranchModified(branchName: string): Promise<boolean>;
+  isBranchModified(branchName: string, baseBranch: string): Promise<boolean>;
   isBranchConflicted(baseBranch: string, branch: string): Promise<boolean>;
```

Comment:
we should flip this in another pr to align with other functions above

Reply:
Agree, this one was bugging me :)